src/sdl_font.cpp

```diff
@@ -279,6 +279,29 @@ CachedTTFFont::CachedTTFFont(
         throw std::runtime_error( TTF_GetError() );
     }
     TTF_SetFontStyle( font.get(), TTF_STYLE_NORMAL );
+
+    ImGuiIO &io = ImGui::GetIO();
+    static const std::array<ImWchar, 17> ranges = {
+        0x0020, 0x052F,
+        0x1D00, 0x1DFF,
+        0x2000, 0x206F,
+        0x20A0, 0x20CF,
+        0x2100, 0x214F,
+        0x2190, 0x22FF,
+        0x2500, 0x27BF,
+        0xC900, 0xC9FF,
+        //0x0020, 0xCFFF,
+        0
+    };
+    imgui_font = io.Fonts->AddFontFromFileTTF( typeface.c_str(), fontsize, nullptr, ranges.data() );
```

Comment:
yeah seems I was missing some unicode ranges for subscripted and superscripted text. i have added those and verified the ⁺ shows up properly. this will probably need to be revisited a few times as we keep discovering new odd character ranges used by CDDA

Reply:
What's preventing us from just supplying all visible unicode character ranges though?

Reply:
I vaguely remember loading all Unicode glyphs in imtui will OOM.